### PR TITLE
[docs-only] Some more details about the tests system, fix #1836

### LINKED ISSUE #1836: Short doc about (acceptance) testing?
I writing tests for the new Nextcloud storage driver. Is there any short doc about the different kind of tests that exist for reva and how they interrelate?

I understand that Reva's [acceptance tests](https://github.com/cs3org/reva/tree/master/tests/acceptance) are a fork of the ownCloud10 core API tests?

What are the litmus tests? Why are there old and new litmus tests?

`make test-acceptance-api` fails with "/bin/bash: /tests/acceptance/run.sh: No such file or directory" on my computer.

### PR DESCRIPTION
**Pull request by michielbdejong**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -102,6 +117,8 @@ If you see `TestGetManagerWithInvalidUser/Nil_in_user` fail, [try removing](http
 6. run the tests
     ```
     cd testrunner
+    # if on Ubuntu, for instance:
+    apt install php-xml php-curl composer
```
Comment (phil-davis) [inline README.md:121]: I wonder where we should put these PHP requirements for the test suite?
This is a one-time command to get this installed on the local system.
Maybe put a note before step 1 in this list?

**Review by phil-davis — APPROVED** [review]
LGTM

**Comment by michielbdejong** [comment]
I found another error in the description of 'unit tests / GRPC tests', now described it as:
* Unit tests
* Integration tests (GRPC)
* Litmus tests (WebDAV)
* Acceptance tests (ownCloud legacy)

Hope this is accurate now.

**Review by individual-it — APPROVED** [review]
I think this is a good description of the current test infrastructure

**Review by labkode — APPROVED** [review]

**Pull request merged**